src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -3992,17 +3992,6 @@ CorInfoIntrinsics ZapInfo::getIntrinsicID(CORINFO_METHOD_HANDLE method,
                                           bool * pMustExpand)
 {
     CorInfoIntrinsics intrinsicID = m_pEEJitInfo->getIntrinsicID(method, pMustExpand);
```

Comment:
Nit: This can be just `return m_pEEJitInfo->getIntrinsicID(method, pMustExpand);` now